### PR TITLE
Make policies importable.

### PR DESCRIPTION
**Pull request by paddycarver**
This makes policies importable. Minor changes were needed for this:

* Read now sets name, in addition to policy.
* We added the Importer functionality and a test for it.
* Our test config needed to be updated to allow the policy name to be
  injected, to help ensure uniqueness across runs.

### CODE REVIEW

**Comment by paddycarver** [comment]
Test failure is an acctest vendoring failure. Should be resolved by #12.

File: vault/import_policy_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccPolicy_importBasic(t *testing.T) {
+	name := "test-" + acctest.RandString(10)
```
Comment (catsby) [inline vault/import_policy_test.go]: There's also [RandomWithPrefix](https://github.com/hashicorp/terraform/blob/master/helper/acctest/random.go#L30), if integers are valid in the name.

File: vault/resource_policy.go [hunk 2]
```diff
@@ -77,6 +80,7 @@ func policyRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	d.Set("policy", policy)
+	d.Set("name", name)
```
Comment (catsby) [inline vault/resource_policy.go]: Should we set this in the `CREATE` method, `policyWrite`? Right now, `policyWrite` does not call `policyRead`, so the `name` attribute won't be set until a `refresh` operation happens. 

Normally we call `create` and it returns `read`. If that's not necessary here, I recommend we set `name` in `policyWrite` then.

File: vault/import_policy_test.go (hunk 1 above)
Comment (paddycarver) [reply vault/import_policy_test.go]: TIL! Thanks :)

File: vault/resource_policy.go (hunk 2 above)
Comment (paddycarver) [reply vault/resource_policy.go]: Good call, I missed that wasn't happening here. Updated to now call `policyRead` from `policyWrite`.

**Pull request merged**